docs/maintainers-guide.md

```diff
@@ -0,0 +1,59 @@
+# Maintainers Guide
+
+This page offers guidance to project maintainers regarding our setup procedures, release processes, package creation, and other related tasks.
+
+# Maintainer Onboarding and Best Practices
+
+## Becoming a Maintainer or Triager
+
+If you are interested in becoming a maintainer, you can join our community. We expect you to be actively involved.
+
+Also, if you're interested in helping managing issues with labels and interacting with incoming requests, you can have a "triager" role!
+
+## Maintainer Responsibilities and Expectations
+
+1. As a maintainer, there is no strict time obligation, as we understand that everyone's ability to commit can fluctuate. However, we do expect maintainers to communicate openly and transparently with the team and the community.
+
+2. As a maintainer, you are expected to uphold a positive and inclusive team culture. This includes following the guidelines outlined in the [Openscapes team culture page](https://openscapes.github.io/series/core-lessons/team-culture.html) and the [recorded psychological safety talk](https://www.youtube.com/watch?v=rzi-qkl8u5M) . By doing so, you can help ensure that all team members and contributors feel safe, respected, and valued.
+
+
+## Maintainer Processes Beyond Regular Contributing
+
+1. As a maintainer, label issues clearly and consistently to help contributors identify issue types and priority. Use 'good first issue' for contributor-friendly issues.
+
+2. As a maintainer, create welcoming environment when communicating with contributors (issue / PR / discussion posters).
+
+3. As a maintainer reviewing and merging contributions is critical. Here are some best practices:
+
+    3a. Review contributions thoroughly.
+
+    3b. Provide constructive feedback.
+
+    3c. Communicate clearly and respectfully.
+
+    3d. Merge contributions promptly.
+
+4. As a maintainer, you will be releasing different versions. More on this in [here](#making-a-release).
```

Comment:
The above two sections are exactly what we need! Thank you :star_struck:

Reply:
Could you please be more specific about this? I'm kind of confused here.

Reply:
Sorry for being unclear. I was saying that I think the "Maintainer Responsibilities and Expectations" & " Maintainer Processes Beyond Regular Contributing" sections look great :)